seed/data_importer/match.py

```diff
@@ -631,8 +681,9 @@ def states_to_views(unmatched_state_ids, org, access_level_instance, cycle, Stat
             merged_between_existing_count += count
             existing_state_ids = [state["id"] for state in sorted(existing_state_matches, key=lambda state: state["updated"])]
             # The following merge action ignores merge protection and prioritizes -States by most recent AuditLog
-            merged_state = merge_states_with_views(existing_state_ids, org.id, "System Match", StateClass)
-            merge_state_pairs.append((merged_state, state))
+            merged_state = merge_states_with_views(existing_state_ids, org.id, "System Match", state_klass)
+            if merged_state:
```

Comment:
This is realted to your other comment about
```
if not views:
   return
```
If the matching states are not attatched to views the `merge_states_with_views` function would return `None`. Im thinking that function should be updated to always return the `merged_state` so this could disappear.